src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -959,6 +960,57 @@ def _configure_security_policy_storage_params(arg_ctx):
         c.argument('allow_data_loss',
                    arg_type=allow_data_loss_param_type)
 
+    ###############################################
+    #             sql instance pool               #
+    ###############################################
+
+    with self.argument_context('sql instance-pool') as c:
+        c.argument('instance_pool_name', options_list=['--name', '-n'],
+                   help="Instance Pool Name")
+        c.argument(
+            'tier',
+            arg_type=tier_param_type,
+            required=True,
+            help='The edition component of the sku. Allowed values: GeneralPurpose, BusinessCritical.')
```

Comment:
Please use get_enum_type() here and delete " Allowed values: GeneralPurpose, BusinessCritical."